src/services/StreamService.ts

```diff
@@ -8,9 +8,9 @@ class StreamService {
      * to a JS object
      * @param event
      */
-    public static getActivitiesStream(event: any) {
+    public static getVisitsStream(event: any) {
         return event.Records.filter((record: DynamoDBRecord) => { // Retrieve "MODIFY" events
-            return record.eventName === "MODIFY";
+            return record.eventName === "MODIFY" && record.dynamodb && record.dynamodb.NewImage && record.dynamodb.NewImage.activityType.S === "visit";
```

Comment:
As on other PR, 
I really hoped to find a good "optional chaining" option for TS, but apparently not. Best option is probably lodash, which is implemented thus:
```
import get from "lodash/get";
...
get(record,"dynamodb.NewImage.activityType.S") === "visit"
```
This should save us unnecessary &&'s, and minimise the bundle size increase.